customresources/s3writer/__init__.py

```diff
@@ -0,0 +1 @@
+#
```

Comment:
Blank files are ignored by some version control systems. It's a fairly long Python tradition to put the minimal # in there since a __init__.py is needed for Python to see the dir as a package.